parameter_files/fates_params_default.cdl

```diff
@@ -983,55 +983,55 @@ data:
     0.8, 0.8, 0.8, 0.8 ;
 
  fates_allom_agb1 = 0.0673, 0.1364012, 0.0393057, 0.2653695, 0.0673, 
-    0.0728698, 0.06896, 0.06896, 0.06896, 0.01, 0.01, 0.01 ;
+    0.0728698, 0.06896, 0.06896, 0.06896, 0.001, 0.001, 0.003 ;
 
  fates_allom_agb2 = 0.976, 0.9449041, 1.087335, 0.8321321, 0.976, 1.0373211, 
-    0.572, 0.572, 0.572, 0.572, 0.572, 0.572 ;
+    0.572, 0.572, 0.572, 1.6592, 1.6592, 1.3456 ;
 
  fates_allom_agb3 = 1.94, 1.94, 1.94, 1.94, 1.94, 1.94, 1.94, 1.94, 1.94, 
-    1.94, 1.94, 1.94 ;
+    1.248, 1.248, 1.869 ;
 
  fates_allom_agb4 = 0.931, 0.931, 0.931, 0.931, 0.931, 0.931, 0.931, 0.931, 
-    0.931, 0.931, 0.931, 0.931 ;
+    0.931, -999.9, -999.9, -999.9 ;
 
- fates_allom_agb_frac = 0.6, 0.6, 0.6, 0.6, 0.6, 0.6, 0.6, 0.6, 0.6, 0.6, 
-    0.6, 0.6 ;
+ fates_allom_agb_frac = 0.6, 0.6, 0.6, 0.6, 0.6, 0.6, 0.6, 0.6, 0.6, 1, 
+    1, 1 ;
 
- fates_allom_amode = 3, 3, 3, 3, 3, 3, 1, 1, 1, 1, 1, 1 ;
+ fates_allom_amode = 3, 3, 3, 3, 3, 3, 1, 1, 1, 5, 5, 5 ;
 
  fates_allom_blca_expnt_diff = -0.12, -0.34, -0.32, -0.22, -0.12, -0.35, 0, 
-    0, 0, 0, 0, 0 ;
+    0, 0, -0.487, -0.487, -0.259 ;
 
  fates_allom_cmode = 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1 ;
 
  fates_allom_d2bl1 = 0.04, 0.07, 0.07, 0.01, 0.04, 0.07, 0.07, 0.07, 0.07, 
-    0.07, 0.07, 0.07 ;
+    0.0004, 0.0004, 0.0012 ;
 
  fates_allom_d2bl2 = 1.6019679, 1.5234373, 1.3051237, 1.9621397, 1.6019679, 
-    1.3998939, 1.3, 1.3, 1.3, 1.3, 1.3, 1.3 ;
+    1.3998939, 1.3, 1.3, 1.3, 1.7092, 1.7092, 1.5879 ;
 
  fates_allom_d2bl3 = 0.55, 0.55, 0.55, 0.55, 0.55, 0.55, 0.55, 0.55, 0.55, 
-    0.55, 0.55, 0.55 ;
+    0.3417, 0.3417, 0.9948 ;
 
  fates_allom_d2ca_coefficient_max = 0.2715891, 0.3693718, 1.0787259, 
     0.0579297, 0.2715891, 1.1553612, 0.6568464, 0.6568464, 0.6568464, 
-    0.6568464, 0.6568464, 0.6568464 ;
+    0.0408, 0.0408, 0.0862 ;
 
  fates_allom_d2ca_coefficient_min = 0.2715891, 0.3693718, 1.0787259, 
     0.0579297, 0.2715891, 1.1553612, 0.6568464, 0.6568464, 0.6568464, 
-    0.6568464, 0.6568464, 0.6568464 ;
+    0.6568464, 0.0408, 0.0862 ;
```

Comment:
@XiulinGao can you confirm that fates_allom_d2ca_coefficient_min for PFT 10 should **not** be changed?  I see that PFT 11 and 12 are changed, and PFT 10 is changed for the max term.